src/util/error.js

```diff
@@ -4,6 +4,8 @@ import root from 'window-or-global';
 export default function error (message) {
   const { DEBUG } = root;
   if (!isUndefined(DEBUG) && DEBUG) {
+    console.error(message);
```

Comment:
What's the rationale here? What are other libs doing in dev mode? Incremental DOM throws. Not sure what React does. I believe they throw, as well.